uiowa/uiowa
# Pharmacy palliative grad map update

Closes #4866 
# How to test


1. Make sure state verbiage shows vs territory (and that the drop down says all instead of any still)
2. The lower right-hand corner of the map: State Information and Click once on any state display
